express-cloudinary/routes/index.js

```diff
@@ -0,0 +1,15 @@
+const express = require('express');
+const router = express.Router();
+
+const postsRouter = require('./posts');
+
+router.use('/posts', postsRouter);
+router.use('/new', postsRouter);
```

Comment:
the new is inside the `posts`route, you should not need to import the `/new` route in this file